packages/react-components/src/components/Input/Input.tsx

```diff
@@ -1,34 +1,90 @@
 import * as React from 'react';
 import cx from 'clsx';
 
+import {
+  VisibilityOn as VisibilityOnIcon,
+  VisibilityOff as VisibilityOffIcon,
+} from '@livechat/design-system-icons/react/material';
+
 import styles from './Input.module.scss';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
 
-type InputSize = 'xsmall' | 'small' | 'medium' | 'large';
+type InputSize = 'compact' | 'medium' | 'large';
 
-export interface InputProps extends React.HTMLAttributes<HTMLInputElement> {
-  size?: InputSize | undefined;
-  error?: boolean | undefined;
-  disabled?: boolean | undefined;
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  wrapperSize?: InputSize;
```

Comment:
Maybe `inputSize`. I'm not sure if `wrapper` is easy to understand.

Reply:
I definitely agree with that. `inputSize` feels much better